### ISSUE TITLE
Not rendering in Safari

### ISSUE THREAD

**Issue by bretonics** [lepture/github-cards#48]
For some reason it has stopped rendering in Safari browser. In all other browsers, it does show up but the background for the themed github card is black.

**Comment by lepture** [comment]
jsdelivr has a bug now. try lab.lepture.com

**Comment by everett-toews** [comment]
I replaced `cdn.jsdelivr.net/github-cards/latest/widget.js` with `lab.lepture.com/github-cards/widget.js` to get it working again.

**Comment by HorlogeSkynet** [comment]
Hey @bretonics @lepture @everett-toews, I've just replaced the URL like proposed above !
The script is now loading again, but the background is black (as pointed out on the first message issue, I suppose) 🙁

Thanks for this project anyway 👋

**Comment by bretonics** [comment]
@HorlogeSkynet, indeed. I suppose that should be a simple switch for the source [here](https://raw.githubusercontent.com/lepture/github-cards/master/jsdelivr/cards/medium.html) and [here](https://github.com/lepture/github-cards/blob/master/theme/medium.css#L93) then...?

@lepture, Why not have the background image in this same repo?

**Comment by lepture** [comment]
@bretonics PR is welcome

**Comment by bretonics** [comment]
@lepture your package is not on npm. Would this not be easier?

**Comment by bretonics** [comment]
I've done some tests using the [new jsDelivr method being implemented](https://github.com/jsdelivr/jsdelivr#github) and it seems that because of the switch from `//cdn.jsdelivr.net/github-cards/latest/widget.js` to what going forward should be `//cdn.jsdelivr.net/gh/lepture/github-cards@latest/jsdelivr/widget.js` causes the issue. 

Since the new method auto-updates, and the old has stopped doing so [ref](https://github.com/jsdelivr/jsdelivr/issues/18007), it seems to me the problem lies in that it tries to find version the latest version, **1.0.3**, in the CDN files stored, but is unable since this file is missing from jsDelivr.

If you try `//cdn.jsdelivr.net/github-cards/1.0.2/widget.js` or  the new method `//cdn.jsdelivr.net/gh/lepture/github-cards@latest/jsdelivr/widget.js`, you get an HTML doc, yet implementing 
`//cdn.jsdelivr.net/gh/lepture/github-cards@latest/jsdelivr/widget.js` results in "Couldn't find the requested release version 1.0.3."
